src/main/java/it/gov/pagopa/arc/service/bizevents/BizEventsServiceImpl.java

```diff
@@ -63,4 +77,31 @@ public Resource retrieveTransactionReceiptFromBizEvents(String transactionId) {
         String retrievedUserFiscalCode = SecurityUtils.getUserFiscalCode();
         return bizEventsConnector.getTransactionReceipt(retrievedUserFiscalCode, transactionId);
     }
+
+    @Override
+    public NoticesListResponseDTO retrievePaidListFromBizEvents(String continuationToken, Integer size, Boolean isPayer, Boolean isDebtor, String orderBy, String ordering) {
+        NoticesListDTO noticesListDTO = NoticesListDTO.builder().build();
+        NoticesListResponseDTO noticesListResponseDTO =  NoticesListResponseDTO.builder().noticesListDTO(noticesListDTO).build();
+        String retrievedContinuationToken = null;
+
+        String retrievedUserFiscalCode = SecurityUtils.getUserFiscalCode();
+
+        BizEventsPaidResponseDTO paidResponseDTO = bizEventsPaidNoticeConnector.getPaidNoticeList(retrievedUserFiscalCode, continuationToken, size, isPayer, isDebtor, orderBy, ordering);
+
+        if(paidResponseDTO != null){
+            if (StringUtils.isNotBlank(paidResponseDTO.getContinuationToken())){
+                retrievedContinuationToken = paidResponseDTO.getContinuationToken();
+            }
+
+            if(!paidResponseDTO.getNotices().isEmpty()){
+                List<NoticeDTO> noticeDTOList = bizEventsPaidNoticeDTO2NoticeDTOMapper.toNoticeDTOList(paidResponseDTO.getNotices());
+                noticesListDTO = bizEventsPaidNoticeListDTO2NoticesListDTOMapper.toNoticesListDTO(noticeDTOList);
+
+                noticesListResponseDTO= bizEventsPaidResponseDTO2NoticesListResponseDTOMapper.toNoticesListResponseDTO(noticesListDTO, retrievedContinuationToken);
```

Comment:
3 mappers to build a result sounds like some inefficient operation has been designed...

also the variable names are confusing... noticeDTOList, noticesListDTO, noticesListResponseDTO...